pages/api/me/password.js

```diff
@@ -55,23 +48,21 @@ async function put(req, res) {
       return;
     } 
 
-    const session = await Session.getSession(session_token);
-    const forcePasswordChange = await People.updatePassword({id: session.people_id, oldPassword: currentPassword, newPassword});
-  
-    req.session.cache = {
-      ...req.session.cache,
-      forcePasswordChange,
-    };
-    await req.session.save();
+    const { sessionToken } = req.user;
```

Comment:
This bit is not backward compatible.

At this point, we assumed all clients would access KlaudSol CMS using JWT tokens. However, the KlaudSol CMS Admin itself uses sessions. We must be able to handle both scenarios.